### PR TITLE
Editor

### PR DESCRIPTION
**Pull request by appinteractive**
Added [Scrumpys TipTap](http://tiptap.scrumpy.io/) editor which is based on [Prosemirror](http://prosemirror.net/).

> Needs backend on branch "editor"
Related with https://github.com/Human-Connection/Nitro-Backend/pull/136

**Todo`s:**
- [x] edit posts
- [ ] create posts
  - [x] title
  - [x] content
  - [ ] tags
  - [ ] categories
- [ ] fix broken integration test as now you cant report your own contributions or your self as a user
- [x] always add `target="_blank"` to the rendered links
- [ ] render all content with the editor-content component
- [ ] **Stylings**
  - [x] bold
  - [x] italic
  - [ ] strike
- [ ] **Elements**
  - [x] links

Embeds are comming later

bildschirmfoto 2019-01-28 um 12 32 03
bildschirmfoto 2019-01-28 um 12 31 28
bildschirmfoto 2019-01-28 um 12 31 20
bildschirmfoto 2019-01-28 um 12 31 15
bildschirmfoto 2019-01-28 um 12 31 04
bildschirmfoto 2019-01-28 um 12 30 57

### CODE REVIEW

File: pages/post/_slug/edit/_id.vue [hunk 1]
```diff
@@ -25,6 +25,9 @@ export default {
       if (!this.Post || !this.Post[0].id) {
         return
       }
+      if (this.Post[0].author.id !== this.$store.getters['auth/user'].id) {
```
Comment (roschaefer) [inline pages/post/_slug/edit/_id.vue]: This is very dirty. Isn't there a VueJS plugin for authorization?

Comment (appinteractive) [reply pages/post/_slug/edit/_id.vue]: No but nuxt middleware. But currently I don’t know if it would work with vue-Apollo

**Comment by roschaefer** [comment]
@appinteractive can you fix CodeFactor please? We can merge afterwards

**Comment by appinteractive** [comment]
> @appinteractive can you fix CodeFactor please? We can merge afterwards

@roschaefer I disabled it, will not be shown on next build. So merging that branch?

**Pull request merged**